### PR TITLE
pegin view success update

### PR DESCRIPTION
**Pull request by hard75**

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by ronaldsg20 — DISMISSED** [review]
LGTM

**Review by josedahlquist — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/components/exchange/TxSummary.vue [hunk 1]
```diff
@@ -241,16 +249,23 @@ export default class TxSummary extends Vue {
...
     } else {
-      this.showTxId = false;
       result = this.VALUE_INCOMPLETE_MESSAGE;
     }
     return result;
   }
 
   get computedRefundAddress(): string {
-    return this.refundAddress !== '' ? this.refundAddress : this.VALUE_INCOMPLETE_MESSAGE;
+    let result;
```
Comment (alexjavabraz) [inline src/components/exchange/TxSummary.vue:262]: You can use:
let result = this.VALUE_INCOMPLETE_MESSAGE;
Without else.

File: src/components/exchange/TxSummary.vue [hunk 2]
```diff
@@ -241,16 +249,23 @@ export default class TxSummary extends Vue {
...
+    if (this.refundAddress !== '0x') {
+      result = this.refundAddress;
+    } else {
+      result = this.VALUE_INCOMPLETE_MESSAGE;
+    }
```
Comment (alexjavabraz) [inline src/components/exchange/TxSummary.vue:267]:
```suggestion
    let result = this.VALUE_INCOMPLETE_MESSAGE;
    if (this.refundAddress !== '0x') {
      result = this.refundAddress;
    }
```

**Review by alexjavabraz — CHANGES_REQUESTED** [review]

File: src/components/exchange/TxSummary.vue (hunk 2 above)
Comment (hard75) [reply src/components/exchange/TxSummary.vue:267]: ok, it's fine

**Review by alexjavabraz — DISMISSED** [review]
LGTM

**Review by alexjavabraz — DISMISSED** [review]
LGTM

**Review by ronaldsg20 — APPROVED** [review]

**Review by ronaldsg20 — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**